packages/utils/src/groupTransactionsByDay.ts

```diff
@@ -0,0 +1,21 @@
+import groupBy from 'lodash.groupby';
+import pickBy from 'lodash.pickby';
+import moment from 'moment';
+import { Statement } from '@deriv/api-types';
+
+const groupTransactionsByDay = (transactions: Statement['transactions']) => {
```

Comment:
Same here.

Reply:
@farzin-deriv discussed this question with @rostislav-deriv and he said he put this function to utils/package because this package has lodash and moment library dependencies which appstore package doesn't have.